lib/bolt/executor.rb

```diff
@@ -68,34 +69,46 @@ def on(nodes, callback = nil)
     end
 
     def run_command(nodes, command)
+      @logger.info("Starting bolt command run #{command}")
       callback = block_given? ? Proc.new : nil
 
       on(nodes, callback) do |node|
-        node.run_command(command)
+        r = node.run_command(command)
+        @logger.info(JSON.dump(r.to_h))
```

Comment:
these should probably have some context. Got result for #{node}: JSON or something

Reply:
to_h is also probably wrong for these we use to_result for json output elsewhere. That should get cleaned up with BOLT-286